app/enterprise/changelog.md

```diff
@@ -887,6 +932,37 @@ keep-alive connections. [7102](https://github.com/Kong/kong/pull/7102)
   - The plugin was not allowing access to any Kong modules within the sandbox except `kong.request`,
     which prevented access to other necessary modules such as `kong.log`.
 
+
+## 2.3.3.3
+**Release Date** 2021/08/12
+
+### Fixes
+
+#### Enterprise
+- Users can now successfully delete workspaces after deleting all entities associated with that workspace.
+  Previously, Kong Gateway was not correctly cleaning up parent-child relationships. For example, creating
+  an Admin also creates a Consumer and RBAC user. When deleting the Admin, the Consumer and RBAC user are
+  also deleted, but accessing the `/workspaces/workspace_name/meta` endpoint would show counts for Consumers
+  and RBAC users, which prevented the workspace from being deleted. Now deleting entities correctly updates
+  the counts, allowing an empty workspace to be deleted.
+- Updates Kong Dev Portal templates' JQuery dependency to v3.6.0, improving security.
+- Users can now include special characters ".", "-", "_", "~" in workspace names. Before when users created
+  workspaces using the UI, the workspace name was validated; however, when using the Admin API to add users,
+  the workspace name was not validated. Because the name was not validated in the Admin Api, users could create workspace names with special characters that would then break the environment.
+- Kong Gateway now escapes "-" and "." special characters in workspace names when building the compiled pattern
+  for collision detection. 
+
+#### Plugins
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  With this fix, when the OpenID Connect plugin is configured with a `config.anonymous` consumer and
+  `config.scopes_required` is set, a token missing the required scopes will have the anonymous consumer
+  headers set when sent to the upstream.
+
+#### Hybrid Mode
+- Control planes are now more lenient when checking data planes' compatibility in hybrid mode. See the
+  [Version compatibility](/enterprise/2.5.x/deployment/hybrid-mode/#version-compatibility)
```

Comment:
```suggestion
  [Version compatibility](/enterprise/2.3.x/deployment/hybrid-mode/#version-compatibility)
```

Not sure if that is correct or not (matched the 2.5.x, but with 2.3.x); feel free to disregard.

Reply:
will update doc to add the version compatibility section since the updates to hybrid mode are now being backported to 2.3.x